kas/platforms/sunxi.yaml

```diff
@@ -25,4 +28,4 @@ repos:
 local_conf_header:
   platform-sunxi: |
     PV_UBOOT_AUTOFDT = "1"
-
+    EXTRA_IMAGE_FEATURES = "debug-tweaks"
```

Comment:
this imo we should make a Kconfig flag that is available for ALL pantavisor-remix targets.

Reply:
For the AUTOFDT OR THE EXTRA_IMAGE_FEATURES ?

Reply:
the extra image features

Reply:
lets make this a text field under the "Make root container" and have default value debug-tweaks...